admin/proc-rhdh-monitoring-logging-aks.adoc

```diff
@@ -0,0 +1,52 @@
+
+[source,bash]
+----
+az aks create/update --resource-group <your-ResourceGroup> --name <your-Cluster> --enable-azure-monitor-metrics
+----
+
+The previous command installs the metrics add-on, which gathers https://learn.microsoft.com/en-us/azure/azure-monitor/essentials/prometheus-metrics-overview[Prometheus metrics]. Using the previous command, you can enable monitoring of Azure resources through both native Azure Monitor metrics and Prometheus metrics. You can also view the results in the portal under *Monitoring -> Insights*. For more information, see https://learn.microsoft.com/en-us/azure/azure-monitor/essentials/monitor-azure-resource[Monitor Azure resources with Azure Monitor].
+
+Furthermore, metrics from both the Managed Prometheus service and Azure Monitor can be accessed through Azure Managed Grafana service. For more information, see https://learn.microsoft.com/en-us/azure/azure-monitor/essentials/azure-monitor-workspace-manage?tabs=azure-portal#link-a-grafana-workspace[Link a Grafana workspace] section.
+
+By default, Prometheus uses the minimum ingesting profile, which optimizes ingestion volume and sets default configurations for scrape frequency, targets, and metrics collected. These settings can be customized through custom configuration. Azure offers various methods, including using different ConfigMaps, to provide scrape configuration and other metric add-on settings. For more information about default configuration, see https://learn.microsoft.com/en-us/azure/azure-monitor/containers/prometheus-metrics-scrape-default[Default Prometheus metrics configuration in Azure Monitor] and https://learn.microsoft.com/en-us/azure/azure-monitor/containers/prometheus-metrics-scrape-configuration?tabs=CRDConfig%2CCRDScrapeConfig[Customize scraping of Prometheus metrics in Azure Monitor managed service for Prometheus] documentation.
+
+== Viewing logs with Azure Kubernetes Services (AKS)
+
+You can perform the following procedure to access live data logs generated by Kubernetes objects and collecting log data in Container Insights within AKS.
+
+.Prerequisites
+
+* You have deployed {product-short} on AKS. For more information, see xref:proc-rhdh-deploy-aks_admin-rhdh[].
+
+.Procedure
+
+View live logs from your {product-short} instance::
++
+--
+. Navigate to the Azure Portal.
+. Browse to the resource group `<your-ResourceGroup>` and locate your AKS cluster `<your-Cluster>`.
+. Select *Kubernetes resources -> Workloads* from the menu.
+. Select the `<your-rhdh-cr>-developer-hub` (in case of Helm Chart installation) or `<your-rhdh-cr>-backstage` (in case of Operator-backed installation) deployment.
+. Click *Live Logs* in the left menu.
+. Select the pod.
++
+NOTE: There must be only single pod.
+
+Live log data is collected and displayed.
+--
+
+View real-time log data from the Container Engine::
++
+--
+. Navigate to the Azure Portal.
+. Browse to the resource group `<your-ResourceGroup>` and locate your AKS cluster `<your-Cluster>`.
```

Comment:
I'm not sure what _Browse to_ means. Is there a more clear word or phrase we can use?